tests/test_common.py

```diff
@@ -72,3 +72,21 @@ def test_version_raise(self):
 def test_normalize_file_permissions():
     assert normalize_file_permissions(0o100664) == 0o100644 # regular file
     assert normalize_file_permissions(0o40775) == 0o40755   # directory
+
+@pytest.mark.parametrize(
+    ("requires_python", "expected_result"),
+    [
+        ("", True),
+        (">2.7", True),
+        ("3", False),
+        (">= 3.7", False),
+        ("<4, > 3.2", False),
+        ('>3.4', False),
```

Comment:
Can we have a test for the pattern people often use to indicate 2.7 or 3.x support, like `>=2.7, !=3.0.*, !=3.1.*, !=3.2.*`.